src/status_im/common/controlled_input/utils.cljs

```diff
@@ -0,0 +1,98 @@
+(ns status-im.common.controlled-input.utils
```

Comment:
Maybe it should be 

status-im.common.utils.controlled-inputs ?

As we already have a folder for just util in some categories

Reply:
We have a bunch of `utils.cljs` under specific topics, so I though this is a dominating pattern. I don't mind, but wdyt?

<img width="760" alt="Screenshot 2024-04-22 at 17 57 55" src="https://github.com/status-im/status-mobile/assets/5786310/f04af8a4-215c-4dcb-924b-a3681379eb35">